import sys

# all is fine
sys.exit(0)

# an error has been detected
# sys.exit(1)